catalog/CatalogDatabaseLite.hpp

```diff
@@ -69,6 +70,16 @@ class CatalogDatabaseLite {
   virtual const CatalogRelationSchema& getRelationSchemaById(const relation_id id) const = 0;
 
   /**
+   * @brief Get a relation by ID.
+   *
+   * @param id The id to search for.
+   * @return The relation with the given ID.
+   **/
+  virtual const CatalogRelation* getRelationById(const relation_id id) const {
```

Comment:
Per [the comment](https://github.com/pivotalsoftware/quickstep/pull/238/files#r64515758), we don't need this method.